### PR TITLE
Workaround firewalld issues

### PR DESCRIPTION
**Pull request by elyezer**
Still using iptables even on RHEL7 in order to avoid Docker issues with
firewalld.

### CODE REVIEW

**Comment by elyezer** [comment]
Added some changes for improving code style.

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -474,6 +479,7 @@ def setup_capsule_firewall():
         ),
     })
 
+
```
Comment (elyezer) [inline automation_tools/__init__.py]: PEP8

File: automation_tools/__init__.py [hunk 2]
```diff
@@ -454,13 +456,16 @@ def setup_capsule_firewall():
...
+            # router
             5647,
-            # Anaconda to download kickstart templates to hosts, and for downloading iPXE firmware
+            # Anaconda to download kickstart templates to hosts, and for
+            # downloading iPXE firmware
             8000,
             # Puppet agent to Puppet master connections
             8140,
-            # Subscription Management Services connection to the reverse proxy for the certificate-based API
+            # Subscription Management Services connection to the reverse proxy
```
Comment (elyezer) [inline automation_tools/__init__.py]: PEP8

File: automation_tools/__init__.py [hunk 3]
```diff
@@ -454,13 +456,16 @@ def setup_capsule_firewall():
             80,
             # Anaconda, yum, Telemetry Services, and Puppet
             443,
-            # The Katello agent to communicate with the Capsule's Qpid dispatch router
+            # The Katello agent to communicate with the Capsule's Qpid dispatch
+            # router
             5647,
-            # Anaconda to download kickstart templates to hosts, and for downloading iPXE firmware
+            # Anaconda to download kickstart templates to hosts, and for
```
Comment (elyezer) [inline automation_tools/__init__.py]: PEP8

File: automation_tools/__init__.py [hunk 4]
```diff
@@ -454,13 +456,16 @@ def setup_capsule_firewall():
             80,
             # Anaconda, yum, Telemetry Services, and Puppet
             443,
-            # The Katello agent to communicate with the Capsule's Qpid dispatch router
+            # The Katello agent to communicate with the Capsule's Qpid dispatch
```
Comment (elyezer) [inline automation_tools/__init__.py]: PEP8

File: automation_tools/__init__.py [hunk 5]
```diff
@@ -407,15 +407,17 @@ def setup_satellite_firewall():
...
-            # Port 80 for HTTP (WWW) must be open to download the bootstrap files.
+            # Port 80 for HTTP (WWW) must be open to download the bootstrap
+            # files.
             80,
             # Port 8140 must be open for incoming Puppet connections with the
             # managed systems.
             8140,
-            # Port 9090 must be open for Foreman Smart Proxy connections with the
-            # managed systems.
+            # Port 9090 must be open for Foreman Smart Proxy connections with
```
Comment (elyezer) [inline automation_tools/__init__.py]: PEP8

File: automation_tools/__init__.py [hunk 6]
```diff
@@ -407,15 +407,17 @@ def setup_satellite_firewall():
             # Port 443 for HTTPS (secure WWW) must be open for incoming
             # connections.
             443,
-            # Port 5671 must be open for SSL communication with managed systems.
+            # Port 5671 must be open for SSL communication with managed
+            # systems.
             5671,
-            # Port 80 for HTTP (WWW) must be open to download the bootstrap files.
+            # Port 80 for HTTP (WWW) must be open to download the bootstrap
```
Comment (elyezer) [inline automation_tools/__init__.py]: PEP8

File: automation_tools/__init__.py [hunk 7]
```diff
@@ -407,15 +407,17 @@ def setup_satellite_firewall():
             # Port 443 for HTTPS (secure WWW) must be open for incoming
             # connections.
             443,
-            # Port 5671 must be open for SSL communication with managed systems.
+            # Port 5671 must be open for SSL communication with managed
```
Comment (elyezer) [inline automation_tools/__init__.py]: PEP8

**Comment by omaciel** [comment]
ACK and roll

**Comment by sthirugn** [comment]
this prevents enabling/starting firewalld in rhel7 host.

workaround: I had to stop iptables, run setup_firewall again with a code change of '>7'

**Pull request merged**